general/project/projectinit.go

```diff
@@ -50,6 +50,13 @@ func (pic *ProjectInitCommand) Run() (err error) {
 	if err != nil {
 		return err
 	}
+	// In case no technologies were detected in the root diretory, try again recursively.
+	if len(technologiesMap) == 0 {
+		technologiesMap, err = coreutils.DetectTechnologies(pic.projectPath, false, true)
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
I suggest moving the following code into a new ```detectTechnologies``` method, to make the code look even better - 
```
	technologiesMap, err := coreutils.DetectTechnologies(pic.projectPath, false, false)
	if err != nil {
		return err
	}
	// In case no technologies were detected in the root diretory, try again recursively.
	if len(technologiesMap) == 0 {
		technologiesMap, err = coreutils.DetectTechnologies(pic.projectPath, false, true)
		if err != nil {
			return err
		}
	}
```